YouVsVirus/Assets/Scripts/levelgethome/CreatePopLevelgethome.cs

```diff
@@ -86,10 +92,19 @@ void Start()
         private void CreateHumans()
         {
             // place player near top left edge of the screen (using the not shrunk screen bounds)
-            Vector3 coords = new Vector3(-randomGridForHumans.screenBounds.x / 0.8f + 0.2f, randomGridForHumans.screenBounds.y / 0.8f - 0.2f, 0f);
+            Vector2 coords;
+            
+            // Create random coordinates and check whether they are spawnable
+            do
+            {
+                coords = new Vector2(-randomGridForHumans.screenBounds.x / 0.8f + 0.2f, randomGridForHumans.screenBounds.y / 0.8f - 0.2f);
+            }
+            while (!nonSpawnableSpaceClass.coordinatesAreSpawnable2D(coords));
+
+            Vector3 coords3D = new Vector3 (coords[0], coords[1], 0);
             //  Place the player
             Player = Instantiate(playerPrefab.GetComponent<Player>(),
-                                 coords,
+                                 coords3D,
```

Comment:
Why do we need the SpawnableSpaceClass when we already know where our player is placed?

Reply:
Ah, ok, old file?